metarecord/views/function.py

```diff
@@ -284,17 +286,41 @@ class FunctionViewSet(DetailSerializerMixin, viewsets.ModelViewSet):
     lookup_field = 'uuid'
 
     def get_queryset(self):
-        queryset = self.queryset.filter_for_user(self.request.user)
+        queryset = self.queryset
 
         if 'version' in self.request.query_params:
             return queryset
```

Comment:
Hmm, isn't it now possible for API users to bypass the user based filtering by using the list endpoint and `version` query param?

Reply:
Actually yes, that should also be added to tests.